source/developers-guide/plugin-quick-start/index.md

```diff
@@ -525,3 +595,520 @@ Possible settings for the parameter `caches` contains each context as constant.
+}
+```
+## Plugin Resources
+
+Plugin meta data and configurations will be configured by using xml files which will be placed like in the example below.
+IDEs like PhpStorm support auto completion by default for these files if schema file location is valid.
+
+```
+SwagSloganOfTheDay
+├──plugin.xml
+├── Resources
+│   ├── config.xml
+│   ├── cronjob.xml
+│   └── menu.xml
+└──SwagSloganOfTheDay.php
+```
+
+You can find the schema of the xml files in `engine/Shopware/Components/Plugin/schema`.
+ - **config.xml:** Defines the plugin configuration form which you can access by the `Basic Settings` or in the detail window of a plugin.
+ - **cronjob.xml:** Defines cronjobs installed with the plugin.
+ - **menu.xml:** Defines new menu items in the backend menu structure of Shopware.
+ - **plugin.xml:** Defines the meta data of your plugin, i.e. label, version, compatibility or the changelog. 
+ 
+<div class="alert alert-warning">
+At the moment it is necessary that the order of the xml elements is equal to the schema file, otherwise you will receive an exception. <br/>
+You can use the CLI to install the plugin with extended error messages: <code>php ./bin/console sw:plugin:install SwagSloganOfTheDay -v</code>
+</div>
+
+## Plugin Metadata 
+ 
+Entires can be separated with semicolons when documenting multiple changes in the changelog.
+
+Example `plugin.xml`:
+ 
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/plugin.xsd">
+    <label lang="de">Slogan des Tages</label>
+    <label lang="en">Slogan of the day</label>
+
+    <version>1.0.0</version>
+    <link>http://example.org</link>
+    <author>shopware AG</author>
+    <compatibility minVersion="5.2.0" />
+
+    <changelog version="1.0.0">
+        <changes lang="de">Farbe geändert; Schriftgröße geändert;</changes>
+        <changes lang="en">changed color; changed font-size;</changes>
+    </changelog>
+</plugin>
+```
+
+## Require other plugins
+If you need other plugins to be installed, then you are able to define them in your plugin.xml file like this
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/plugin.xsd">
+    ...
+    <changelog version="1.0.0">
+        ...
+    </changelog>
+    <requiredPlugins>
+        <requiredPlugin pluginName="SwagRequiredPlugin" minVersion="1.2.0" maxVersion="3.5.2"/>
+    </requiredPlugins>
+</plugin>
+```
+
+## Plugin Configuration / Forms
+
+Backend plugin configuration can be extended by `Resources/config.xml` file. This replaces the usage of `$this->Form()` on old `Shopware_Components_Plugin_Bootstrap`.
```

Comment:
do not mention the old plugin system

Reply:
@swDennis there is still the hint to the old plugin system. remove the sentence

`This replaces the usage of $this->Form() on old Shopware_Components_Plugin_Bootstrap.`

Reply:
D